dolphinscheduler-dist/release-docs/licenses/LICENSE-lombok.txt

```diff
@@ -0,0 +1,104 @@
+Copyright (C) 2009-2021 The Project Lombok Authors.
```

Comment:
I found the scope of `lombok` is `provided`, Should we add lombok license file? cc @kezhenxu94 @ruanwenjun

Reply:
Fixed